kie-server-parent/kie-server-tests/pom.xml

```diff
@@ -647,7 +647,7 @@
                 </container>
                 <configuration>
                   <properties>
-                    <cargo.jvmargs>-Xmx1024m</cargo.jvmargs>
+                    <cargo.jvmargs>-Xmx2048m</cargo.jvmargs>
```

Comment:
This would be worth checking locally if it is needed for all containers. Wdyt @sutaakar?

Reply:
Yes, worth investigating

Reply:
I have analyzed a heap dump in case 1024 MB are used and org.drools.dynamic.DynamicProjectClassLoader takes 457 MB when OOM occurs.
@lucamolteni Is it expected for executable model to require so much of memory?

When I run similar amount of tests on latest master version then org.drools.dynamic.DynamicProjectClassLoader takes 42 MB.